src/locales/en-us/messages.json

```diff
@@ -25,14 +25,9 @@
     "description": "A title of an inflections browser panel",
     "component": "Panel"
   },
-  "TOOLTIP_MOVE_PANEL_LEFT": {
-    "message": "Move Panel to Left",
-    "description": "tooltip for moving the panel to the left",
-    "component": "Panel"
-  },
-  "TOOLTIP_MOVE_PANEL_RIGHT": {
-    "message": "Move Panel to Right",
-    "description": "tooltip for moving the panel to the right",
+  "TOOLTIP_SWAP_PANEL_POSITION": {
```

Comment:
It is only my thoughts - but may be it would be better to save previous exact tooltips and change the message according to the current icon state?

Reply:
That's a very good idea! I think it will be much more informative to users. Will implement it